README.md

```diff
@@ -1,70 +1,45 @@
-# Getting Started with Create React App
+# CoviData Metrics Webapp
+This project is a mobile web applicaation used to check a list of COVID-19 metrics.
```

Comment:
- [ ] You have not used [the correct gitflow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md) as provided in the project requirements. Kindly fix this.